src/main.rs

```diff
@@ -40,22 +55,42 @@ enum BackendArgs {
     WASM,
 }
 
+#[derive(Clone, Copy, Debug, ValueEnum)]
+enum EmitArgs {
```

Comment:
Having the option of `None` here allows for something like `--emit=none,c` which is slightly strange. I think we could probably go for something like
```
emit: Option<Vec<EmitArgs>>
```
and remove the `None` option from the `EmitArgs` enum.

What do you think?